code/data_structures/linked_list/linked_lists_operations/linked_list_cycle.cpp

```diff
@@ -32,33 +31,44 @@ int linked_list_operations(int ch,int x)
         head=n;
         break;
     case 2://Cycle Placement at position=x
-        if(head!=NULL)
+        if(head!=nullptr)
         {
-        for(temp1=head;temp1->next!=NULL;temp1=temp1->next);
-        for(temp2=head;x>1;x--,temp2=temp2->next);
-        temp1->next=temp2;
+	    for(temp1=head;temp1->next!=nullptr;temp1=temp1->next);
+            for(temp2=head;x>1;x--,temp2=temp2->next);
+            temp1->next=temp2;
         }
         break;
     case 3://Cycle Check
         x=0;
         temp1=temp2=head;
-        while(temp2!=NULL&&temp2->next!=NULL)
+        while(temp2!=nullptr&&temp2->next!=nullptr)
         {
             temp1=temp1->next;
             temp2=temp2->next->next;
-            if(temp2->next->next==head)
-            {temp2->next->next=NULL;break;}//Transforms Circular List to Linear List
-            if(temp2->next==head)
-            {temp2->next=NULL;break;}//Transforms Circular List to Linear List
-            if(temp1==temp2){x=1;break;}//Cycle Detection
+            if(temp2->next->next==head)//Transforms Circular List to Linear List

```

Comment:
Very irregular indentation in lines 48 to 62 for the braces.

Reply:
actually indentation using npp and github caused this mix, in my npp this is showing alright, can u suggest some software which is consistent with github format?

Reply:
I can't really say. I would attempt using a different editor to see it.

Reply:
Ok, there seems to be some glitch within Github, I looked at the file on your fork, and it looks perfectly alright.
![screen shot 2017-10-20 at 8 14 16 am](https://user-images.githubusercontent.com/30810402/31820306-b8312e9e-b56e-11e7-9450-967fe76c43cf.png)

Reply:
I wonder what the problem is (or if there is one at all ;P)